### ISSUE TITLE
Index and display (with hyperlinks) MARC 700 fields and all subfields

### ISSUE THREAD

**Issue by seanluyk** [ualbertalib/discovery#707]
It would be really useful for performing arts related searching, to have **added entry personal name fields**(MARC 700) indexed, displayed (and ideally, clickable). This functionality currently exists in iLink, and to me falls under the category of  "functionality we like in iLink that should be replicated in Blacklight"). 

The use case applies to recordings mostly, where there's a corporate author or it's a compilation, and the user would like to discover a particular author/work. Uniform title covers this for single work scores (i.e. not anthologies), but added entries for personal names allow users to search for particular works contained within anthologies, and is the only way to do this with recordings.

Compare this [Blacklight record](https://search.library.ualberta.ca/catalog/5946283) with this [iLink record](http://www.library.ualberta.ca/permalink/opac/5946283/WUAARCHIVE). The added entries for personal names (with subfields added for particular works) allow the user to easily locate other manifestations of these works. 

@theLinkResolver  - does this make cataloguing sense? Am I missing any fields where this is applied?

**Comment by theLinkResolver** [comment]
@seanluyk What you are describing makes sense. The ideal behaviour would be for the link to execute an author search for that phrase (provided that the author search looks in the 100, 110, 111, 700, 710, 711, and 720 fields - @redlibrarian is that what author search does?).

**Comment by theLinkResolver** [comment]
It's implicit in the title of this issue but I need to note that the 700 $d is not currently displayed, but should be, like it is for the 100. (This has come up in conversations with Bib colleagues, and I want to acknowledge that.)

**Comment by redlibrarian** [comment]
The only obstacle for me is mapping MARC fields to the appropriate Solr fields which, in the way Blacklight was designed, is not particularly intuitive. There are currently 6 author fields in Solr: author_display, author_vern_display, author_unstem_search, author_addl_unstem_search, author_t, author_addl_t, some of which are clearly for main authors, some for additional. We'll just have to be careful how we add the additional fields and subfields that we map them to the right Solr fields for index & search. Again, not a problem, just non-trivial and will require some extra thought. / @seanluyk @theLinkResolver

**Comment by theLinkResolver** [comment]
@redlibrarian Indeed.. and I hope I can continue to help with that process in the new year. As we have been adding little bits and pieces of data, I hope we can step back in a future phase and take a more in-depth look at 'chunks' of the records, such as related authors, notes, related titles, etc., both in detail (looking for missing/extraneous pieces and issues of indexing, display and/or clickability) but also holistically (visual arrangement, duplication, etc.).

**Comment by redlibrarian** [comment]
@theLinkResolver @seanluyk and I will get together to work through these fields.

**Comment by redlibrarian** [comment]
I've linked author_addl_t, which should help. In conjunction with #670, I need to work through this in more detail.

**Comment by theLinkResolver** [comment]
Shared by a Bib colleague:

7443586
700 field contains: |iContinuation of (work):|aRojas Osorio, Carlos.|tEstética filosófica en Latinoamérica.
where subfield i explains the relationship between the work in subfield t and the work described in the record.

This record is not in search-test so I can't see the latest version of how it displays (and can't readily find another example). To be revisited in summer.

**Comment by theLinkResolver** [comment]
Sample record: https://search.library.ualberta.ca/catalog/8782575

The occurrence of "Additional authors/performers" is actually because the 700 field here is presenting a relationship to another work (a name-title entry, we used to call it). Put another way, the 700 does different things than just listing additional authors for the thing in question. While this issue isn't being worked on at the moment, I just wanted to drop in a fresher example for future reference!
